hnn_core/network.py

```diff
@@ -251,8 +251,58 @@ def _add_external_feeds(self):
         unique_keys = sorted(self.p_unique.keys())
         self.feedname_list += unique_keys
 
-    def _create_gid_dict(self):
-        """Creates gid dicts."""
+        self._update_gid_dict()  # add feeds -> update the gid_dict
+        self.instantiate_feeds()
+
+    def instantiate_feeds(self):
```

Comment:
docstring missing for public method

Reply:
made it private, but added docstring anyway ;)